sandbox/postman/Published-Cohort-Definitions-API-Sandbox.postman_collection.json

```diff
@@ -0,0 +1,1201 @@
+				}
+			]
+		}
+	],
+	"event": [
+		{
+			"listen": "prerequest",
+			"script": {
+				"type": "text/javascript",
+				"exec": [
+					""
+				]
+			}
+		},
+		{
+			"listen": "test",
+			"script": {
+				"type": "text/javascript",
+				"exec": [
+					""
+				]
+			}
+		}
+	],
+	"variable": [
+		{
+			"key": "SandboxURL",
+			"value": "https://internal-dev-sandbox.api.service.nhs.uk/caas-published-cohort-definitions/api",
```

Comment:
Changing this to https://internal-qa-sandbox.api.service.nhs.uk/caas-published-cohort-definitions/api, which is the released sandbox URL, gets rid of the 503 issues we've been having. I think it's worth swapping them over.

Reply:
✅